File: tests/fixtures/python_sample/animals.py
class Animal:
    """Base class for all animals."""

    def __init__(self, name: str) -> None:
        self.name = name

    def speak(self) -> str:
        """Make the animal speak."""
        return ""


class Dog(Animal):
    """A dog."""

    def speak(self) -> str:
        return f"{self.name} says woof"

File: tests/fixtures/go_sample/animals.go
package animals

// Animal represents a generic animal.
type Animal struct {
	Name string
}

// Speak makes the animal produce a sound.
func (a *Animal) Speak() string {
	return ""
}

// Dog is a dog.
type Dog struct {
	Animal
}

// Speak makes the dog bark.
func (d *Dog) Speak() string {
	return d.Name + " says woof"
}
